node/src/accountant/scanners/mod.rs

```diff
@@ -529,18 +520,12 @@ impl PayableScanner {
         let existent_and_nonexistent = self
             .pending_payable_dao
             .fingerprints_rowids(&hashes_of_failed);
-        let existent = existent_and_nonexistent
-            .rowid_results
-            .into_iter()
-            .collect_vec();
-        let nonexistent = existent_and_nonexistent
-            .no_rowid_results
-            .into_iter()
-            .collect_vec();
-        let missing_fgp_err_msg_opt =
-            err_msg_if_failed_without_existing_fingerprints(nonexistent, serialize_hashes);
-        if !existent.is_empty() {
-            let (ids, hashes) = separate_rowids_and_hashes(existent);
+        let missing_fgp_err_msg_opt = err_msg_if_failed_without_existing_fingerprints(
```

Comment:
Regarding `err_msg_if_failed_without_existing_fingerprints`, if we rename it to something like `err_msg_for_failure_with_expected_but_missing_fingerprints`